src/app/scripts/cac/pages/cac-pages-map.js

```diff
@@ -154,8 +154,26 @@ CAC.Pages.Map = (function ($, Handlebars, _, moment, MapControl, Routing, MockDe
         var $container = $('<div></div>').addClass('destinations');
         $.each(destinations, function (i, destination) {
             var $destination = $(CAC.Map.Templates.destinationBlock(destination));
-            $destination.click(function (event) {
-                // TODO: What to do on click?!?!
+
+            $destination.click(function () {
```

Comment:
I didn't know about it when I wrote the original TODO here, but there is an intermediate view for the sidebar where on click we show the title, image and full description for a destination. The mockup is in the wireframes. 

Either way, we'll need this code because there is a button on that intermediate view that is for getting directions.